pcdsdevices/ladm.py

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/python
+
+import logging
+
+import numpy as np
+from ophyd import Component as Cpt
+from ophyd.signal import EpicsSignal
+from prettytable import PrettyTable
+
+from pcdsdevices.epics_motor import IMS
+
+from .device import GroupDevice
+from .interface import BaseInterface
+
+logger = logging.getLogger(__name__)
+
+alpha_r = 63 * np.pi/180  # rail angle
+r = 2960.0  # first rail distance from sample to rail rod at 27deg
+R = 6735.0
+alpha_cos = np.cos(27 * np.pi/180)
+
+
+def ThetaToMotors(theta, samz_offset=0):
+    theta = theta * np.pi/180  # change to radian
+    x1 = (r) * np.sin(theta)/(np.sin(alpha_r) * np.sin(alpha_r+theta))
+    x2 = (R) * np.sin(theta)/(np.sin(alpha_r) * np.sin(alpha_r+theta))
+    dz = (r)/np.sin(alpha_r)-x1 * np.sin(alpha_r)/np.sin(theta)
+    if theta == 0:
+        dz = 0
+    return x1, x2, dz
+
+
+def y1ToGamma(y1):
```

Comment:
Why are these defined outside the LADM class?  They seem to be rather specific to the geometry of the LADM, with coordinate names that only someone familiar with the LADM would understand.  

Generally I dislike such opaque naming.  Without a diagram it's impossible to verify that this math is doing the right thing